BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -0,0 +1,76 @@
+#include "BSP_Timer.h"
+
+static void (*Timer1Handler)(int);
+static void (*Timer2Handler)(int);

```

Comment:
Could you pass void instead of int? If so, that would be preferable.

Reply:
Have you tried that? So it would be defined like:

`static void (*timer1Handler)(void)`
and the same for the second handler